apply/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang=en>
+	<head>
+		<title>How to Apply</title>
+		<link rel="shortcut icon" type="image/png" href="../resources/favicon.png">
+		<link rel="stylesheet" type="text/css" href="../resources/stylesheet.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta name="Description" content="How to apply to The Forum Helpers">
+
+		<!-- Global site tag (gtag.js) - Google Analytics -->
+		<script async src="https://www.googletagmanager.com/gtag/js?id=UA-197061964-2"></script>
+		<script>
+		window.dataLayer = window.dataLayer || [];
+		function gtag(){dataLayer.push(arguments);}
+		gtag('js', new Date());
+
+		gtag('config', 'UA-197061964-2');
+		</script>
+	</head>
+
+	<body>
+		<header id="header"></header>
+
+		<div class="applyContent">
+		<br>
+			<h1>How to apply to The Forum Helpers</h1>
+			<p>Status: Applications are <span style="color: red">closed</span>, check back soon!</p>
+			<!--<p>Status: Applications are <span style="color: #1bc900">open</span></p>-->
+			<h2>Introduction</h2>
+			<p>
+				Thank you for your interest in applying to The Forum Helpers! We are always on the lookout for new members! Please ensure that you take the time to thoroughly read the application process below so that your application can be considered.
+			</p>
+
+			<h2>Requirements</h2>
+			<p>
+				The requirements to join The Forum Helpers exist to ensure that our members are ones that are dedicated to helping out on the forums, and that all of our members follow the community guidelines on and off the forums. The requirements to apply are:
+				<ul>
+					<li>You must have 500+ posts on the Scratch Discussion Forums.</li>
+					<li>You will not be added if any of your posts made within the month contain spam, trolling, necroposts, or generally breaking the rules.</li>
+					<li>If you are mainly active in Collaborations, Requests, Project Ideas, Things I'm Making and Creating, Things I'm Reading and Playing, New Scratchers, or Announcements, you may not apply unless you are active in other forums also.</li>
+				</ul>
+			</p>
+
+			<h2>I fit the requirements; how do I apply?</h2>
+			<p>
+				If you fit the requirements above, you are able to apply to The Forum Helpers. To apply, please fill out the following application and post it in <a class="FHULink" href="https://scratch.mit.edu/studios/3688309/comments/">our main studio</a>.
+			</p>
+				<ol>
+				<li> Which forums are you the most active in?</li>
+				<li>Provide links to 2 – 4 posts you made that you find the most constructive. (NOTE: They must NOT be a Sticky, iTopic, or any kind of guide)</li>
+				<li>Provide a link to your most recent forum post.</li>
+			</ol>
+			<p>Things to keep in mind when submitting your application. Not following these rules could result in your application being declined:</p>
+			<ul>
+				<li>Please do not use any third party service (eg. Ocular) to share deleted posts.</li>
```

Comment:
minor thing.. ocular is spelled with a lowercase o.

Reply:
Thanks!